src/PackageManagement/NuGetPackageManager.cs

```diff
@@ -1156,6 +1156,17 @@ private void InitializePackagesFolderInfo(string packagesFolderPath)
 
             sources.UnionWith(enabledSources);
 
+            // Read the current lock file if it exists
+            LockFile originalLockFile = null;
+            var lockFileFormat = new LockFileFormat();
```

Comment:
I keep meaning to make this static... there's no instance state in `LockFileFormat`